internal/generator/vector/helpers/helpers.go

```diff
@@ -30,3 +30,15 @@ func TrimSpaces(in []string) []string {
 func FormatComponentID(name string) string {
 	return strings.ToLower(Replacer.Replace(name))
 }
+
+func Unique(stringSlice []string) []string {
```

Comment:
There is kube struct called "set" which can replace this.  I believe I saw its use in our api package for Reserved names